gc/base/standard/ConcurrentGC.hpp

```diff
@@ -374,6 +374,16 @@ class MM_ConcurrentGC : public MM_ParallelGlobalGC
 		return ret;
 	}
 
+	MMINLINE virtual bool acquireExclusiveVMAccessForCycleStart(MM_EnvironmentBase *env)
```

Comment:
for symmetry (but I could also argue that UI variant is even more special in a sense it does a GC STW increment without need for flushing, what is naively always needed) let's make this method pure virtual in base, and move the implementation to UI